src/Components/App/App.js

```diff
@@ -41,7 +54,9 @@ class App extends Component {
           </header>
         </nav>
         <main className="main-section">
-           {this.state.movieInfo ? <MovieDetails movie={this.state.movieInfo} goToMainView={this.goToMainView}/> : <Movies movies={this.state.movies} getMovieInfo={this.getMovieInfo}/>}
+          {!this.state.movies.length ? <h2>Loading Movies...</h2> : <Movies movies={this.state.movies} getMovieInfo={this.getMovieInfo} getMovieTrailer={this.getMovieTrailer}/>}
```

Comment:
Great loading handling message, but how to modify the loading message while we display the main card view ?

Reply:
You could consider making a `Loader` component!